### ISSUE TITLE
Lockpick macro ignoring reliable talent

### ISSUE THREAD

**Issue by Dprime84** [Saibot393/LocknKey#100]
Hi, i've been messing about with the lockpick macro and noticed that it ignores the reliable talent flag. Is there any way to address that?

**Comment by Saibot393** [comment]
I am afraid you will have to do that manually. To do that, you can go into the lock & key settings of actors prototype token, there you can set "Lockpick formula" setting for that token to `max(1d20,10) + @actor.system.abilities.dex.mod + @actor.system.tools.thief.prof.flat + @actor.system.tools.thief.bonus` and check the "Override Lockpick formula" setting.

**Comment by Dprime84** [comment]
That's exactly what I needed, thank you! Bit of an unrelated question but is there any reason why the pickpocket macro would cause a merchant to stop being interactive? The macro worked correctly and b allowed the player to steal from the merchant but afterwards the npc lost the merchant tag. I put it back in place but the player could no longer interact with that particular token.

**Comment by Saibot393** [comment]
Thats strange, i will look into it, i assume that that was a item piles merchant?

**Comment by Dprime84** [comment]
Yup. A night hag set up as potion vendor. Strangely enough, the player had the option to pick pocket her claw attack...

**Comment by Saibot393** [comment]
Oh, yeah, there is no filter for item types in the pickpocketing. I was planning an adding more options to pickpocketing soon (including type filters and per item dcs), but didn't yet have time to do so.

**Comment by Dprime84** [comment]
Ooh those look like features that would be really nice. Anyhow, I think I'll take the pickpocket macro away for now so I don't have this issue.

**Comment by Saibot393** [comment]
I can not seem to replicate this bug, could you send me the merchants actor .json export, maybe there is a specific setting that causes this

**Comment by Dprime84** [comment]
Got limited access to it right now but I think it might be something to do with the item piles add-on not working correctly with v12

**Comment by Saibot393** [comment]
Yes, that could explain it, i haven't yet tested item piles in v12, was waiting for an official update

**Comment by Saibot393** [comment]
From my testing everything should now work correctly with the newest versions of Lock & Key and Item Piles
